boards/arm/nucleo_g474re/board.cmake

```diff
@@ -1,4 +1,4 @@
 # SPDX-License-Identifier: Apache-2.0
-board_runner_args(pyocd "--target=stm32g474retx")
+board_runner_args(pyocd "--target=stm32g474rbtx")
```

Comment:
This would deserve a comment so we know this is made on purpose, to solve a flashing issue.
Also, are we sure this has no impact otherwise (which should be advertised as limitations )?

Reply:
adding a comment

Reply:
adding a comment